### PR TITLE
addpkg: x86_64-linux-gnu-binutils 2.43-1

### PR DESCRIPTION
**Pull request by kxxt**
Based on aarch64-linux-gnu-binutils.

### CODE REVIEW

File: x86_64-linux-gnu-binutils/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Contributor: Levi Zim <rsworktech@outlook.com>
+# Contributor: Anatol Pomozov <anatol.pomozov@gmail.com>
+
+_target=x86_64-linux-gnu
+pkgname=$_target-binutils
+pkgver=2.43
+pkgrel=1
+pkgdesc='A set of programs to assemble and manipulate binary and object files for the ARM64 target'
```
Comment (Xeonacid) [inline x86_64-linux-gnu-binutils/PKGBUILD:8]:
```suggestion
pkgdesc='A set of programs to assemble and manipulate binary and object files for the x86_64 target'
```

File: x86_64-linux-gnu-binutils/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+build() {
+  cd binutils-$pkgver
+
+  if [ "${CARCH}" != "i686" ];
+  then
+    # enabling gold linker at i686 makes the install fail
+    enable_gold='--enable-gold'
+  fi
```
Comment (Xeonacid) [inline x86_64-linux-gnu-binutils/PKGBUILD:32]: May consider dropping the branch?

File: x86_64-linux-gnu-binutils/PKGBUILD (hunk 1 above)
Comment (kxxt) [reply x86_64-linux-gnu-binutils/PKGBUILD]: Fixed

File: x86_64-linux-gnu-binutils/PKGBUILD (hunk 2 above)
Comment (kxxt) [reply x86_64-linux-gnu-binutils/PKGBUILD]: Thanks for the advice. I have dropped it.

**Pull request merged**